app/javascript/components/validation/ValidationMessage.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { supportEmailLink } from 'lib/error-utils'
+import _capitalize from 'lodash/capitalize'
+import pluralize from 'pluralize'
+
+/** Help users get reach us for support */
+function SupportMessage({ studyAccession }) {
+  return (
+    <div>
+    Need help? Email us at {supportEmailLink} with the errors,
+    your study accession ({studyAccession}),
+    and a description or attachment of your file
+    </div>
+  )
+}
+
+const refresh =
+  <a href="" data-analytics-name="sync-validation-refresh">
+    Refresh the page
+  </a>
+
+/** Summarize errors and warnings */
+function Summary({ msgs, issueCategory, fileName=null, isSync=false }) {
+  // E.g. "Errors", "Warning"
+  const issueTxt = pluralize(_capitalize(issueCategory), msgs.length)
+
+  if (issueCategory === 'error') {
+    const redo = isSync ? refresh : 'Re-choose the file'
+    return <p>{issueTxt} found.  {redo} after correcting {fileName}.</p>
+  } else {
+    return <p>{issueTxt}:</p>
+  }
+}
+
+/**
+ * Displays result of file validation issue for sync UI
+ *
+ * TODO (SCP-4119): Integrate this component into FileUploadControl.js
+ */
+export default function ValidationMessage({
+  studyAccession, errorMsgs, warningMsgs, fileName, isSync=false
```

Comment:
Renaming `isSync` to `showRefreshLink` would be clearer about what the parameter does